Train-In-Blocks-Ltd/Landing
# feat(home): Compare section shows competitor prices.

# TIB Landing PR

## Changes

A brief list of changes

- Added competitor pricing to the home compare section
- Made responsive

## Test Cases

What should the reviewer do and expect to happen when testing

1. Check design
2. Check responsiveness

## Demo

A screenshot or recording of the change.

<img width="1632" alt="Screenshot 2022-06-17 at 19 18 27" src="https://user-images.githubusercontent.com/63963445/174355755-d5fa620f-b1d4-4541-895c-ab3b8937f06b.png">
<img width="954" alt="Screenshot 2022-06-17 at 19 18 33" src="https://user-images.githubusercontent.com/63963445/174355757-239cfe97-2dc8-4c64-8a01-ebf32a9cfc40.png">
<img width="773" alt="Screenshot 2022-06-17 at 19 19 12" src="https://user-images.githubusercontent.com/63963445/174355761-16d6070e-8362-45df-8a26-b63b8d6593f8.png">
